systems/controllers/CMakeLists.txt

```diff
@@ -64,5 +64,9 @@ if (MATLAB_FOUND)
   include_directories (${CMAKE_SOURCE_DIR}/util )
   target_link_libraries(individualCentersOfPressuremex drakeRBM drakeUtil drakeControlUtil)
 
+  add_mex(bodySpatialMotionPDmex bodySpatialMotionPDmex.cpp)
+  include_directories (${CMAKE_SOURCE_DIR}/util )
```

Comment:
don't need this again (include_directories will work for the rest of the file and children)